documents/description.md

```diff
@@ -2,8 +2,26 @@
 
 ## Motivation
 
+*Why is this project useful or interesting, what problem are you trying to address, and why is a DSL an appropriate solution?*
+
+Given how popular electronic music has become, I imagine that several fans probably have a few of their own novel ideas, but no outlet for them. I know personally that most existing audio mixing software is not only very pricey, but also extremely complicated. Simply learning how to use the basics requires several hour long tutorials on Youtube. This project plans to help amateurs who simply want some exposure to audio mixing without the time or capital investments. If designed effectively, audio mixing can be made more user-friendly. Most audio mixing software emulates a DJ console (the thing people use with a lot of knobs and dials and what not). I felt audo mixing can be more effectively done through a well-designed language because ideally it will allow users to translate something they understand in nature language (increase volume, distort by x degree) and apply it to a particular piece of music. The idea is to make the learning curve necessary to edit music less intimidating. 
+
 ## Language domain
 
+*What is the domain that this language addresses, and why is the domain useful? Who will benefit from this language? Are there any other DSLs for this domain? If so, what are they, and how might they influence your language design and implementation?*
+
+The domain addressed by this language is anyone who wants to edit or remix music but lack the necessary tools and experience. This project is mostly targetted at people who have a general interest in audio mixing, but have neither the background nor the expensive software typically used (e.g. Ableton). Additionally, for the less serious music enthusiast, this DSL would be a significantly lighter weight and cheaper alternative to currently existing music software. This DSL would help users learn more about mixing music by providing them basic and easy to understand tools to manipualte sounds as well as allowing them to develop their own tools if necessary. For instance, some built-in features of the language might be basic volume adjustments at specified intervals or allowing for a certain segment of a sound to be looped.
+
+The first notable/complete DSL that comes to mind in this domain would probably be [Audacity](http://www.audacityteam.org/) in that it allows users to edit music relatively easily. However, one key distinction is that this project will be mostly focused on the langauge design and less on the gui interface. With effective langauge design, I hope to take key features from Audacity and make them even easier to understand and to be applied to music through this project. As aforementioned, audio effects can also be taken from Ableton, [VirtualDJ](http://www.virtualdj.com/), among other existing music mixing software.
+
```

Comment:
What specific features from Audacity are difficult, and how might a language make them easier?